middlewares/campaigns.js

```diff
@@ -36,6 +36,7 @@ exports.registerUser = async (req, res) => {
         });
 
         // TODO: add uniqueness check; transliterate gsuiteEmail in case of umlauts or other special characters
+        // TODO: probably move this elsewhere since not all MyAEGEE users need a GSuite account
         const gsuiteEmail = req.body.first_name + '.' + req.body.last_name + '@' + constants.GSUITE_DOMAIN;
         const payload = {
```

Comment:
better to do it this way: have a file to contain all gsuite functions (or even better, make it a class, like GSuiteClient, create instance of it and assign it to something like req.gsuiteClient), then you can reference this class instead of having this logic in a middleware itself

Reply:
That's a great idea